helm/druid/README.md

```diff
@@ -25,20 +6,19 @@
 
 To install the Druid Chart into your Kubernetes cluster :
 
-```bash
-helm install --namespace "druid" --name "druid" incubator/druid
-```
+1. `cd` to root directory of druid project
+2. execute command ```bash helm install --namespace "druid" --name "druid" helm/druid```
```

Comment:
```suggestion
2. execute command `helm install --namespace "druid" --name "druid" helm/druid`
```

Reply:
Thanks for review! By the way, should I just commit suggestion here, or in my local git repository and then force rebase it to one commit?

Reply:
I recommend the former, you can also use your local repo, but we shouldn't force rebase. FYI, https://github.com/apache/druid/blob/master/CONTRIBUTING.md#if-your-pull-request-shows-conflicts-with-master